<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Livro</class>
 <widget class="QWidget" name="Livro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Livros</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLineEdit" name="caixa_titulo">
      <property name="placeholderText">
       <string>Título</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="caixa_editora">
      <property name="placeholderText">
       <string>Editora</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="caixa_autor">
      <property name="placeholderText">
       <string>Autor</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="caixa_isbn">
      <property name="placeholderText">
       <string>ISBN</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="caixa_edicao">
      <property name="placeholderText">
       <string>Edição</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="caixa_volume">
      <property name="placeholderText">
       <string>Volume</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="caixa_paginas">
      <property name="placeholderText">
       <string>Páginas</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="caixa_ano">
      <property name="placeholderText">
       <string>Ano</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="botao_voltar">
      <property name="text">
       <string>Voltar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="botao_confirmar">
      <property name="text">
       <string>Confirmar Adição</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="imagem_livros">
      <property name="minimumSize">
       <size>
        <width>96</width>
        <height>96</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>96</width>
        <height>96</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-repeat: no-repeat;
background-image: url(:/livros/login_livros.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../img/img.qrc"/>
 </resources>
 <connections/>
</ui>
